Classes/shared/SQKCoreDataOperation/SQKCoreDataOperation.m

```diff
@@ -34,6 +34,13 @@ - (instancetype)initWithContextManager:(SQKContextManager *)contextManager
     return self;
 }
 
+- (void)dealloc
+{
+    [[NSNotificationCenter defaultCenter] removeObserver:self
```

Comment:
>In OS X 10.11 and iOS 9.0 NSNotificationCenter [...] observers are not required to un-register in their deallocation method.
But we're not >= iOS 9, yet, right? 
https://developer.apple.com/library/mac/releasenotes/Foundation/RN-Foundation/index.html#10_11NotificationCenter